internal/profile/profile.go

```diff
@@ -76,7 +76,7 @@ func newProfileFromExistingFiles(elasticPackagePath string, profileName string,
 			// if we're treating missing files as soft errors,
 			// just continue on IsNotExist
```

Comment:
Please adjust this comment as it isn't `IsNotExist` anymore.

Reply:
Yes Sure